apps/mull-ui/src/constants.ts

```diff
@@ -14,7 +14,7 @@ export const ROUTES = {
   MESSAGES: '/messages',
   GROUPCHAT: { url: '/messages/group-chat', displayName: 'Group Chat' },
   ANNOUNCEMENTS: { url: '/messages/announcements', displayName: 'Announcements' },
-  PROFILE: '/profile',
+  PROFILE: { profile: '/profile', editProfile: '/profile/edit' },
```

Comment:
We might need a talk about how we are defining our routes. For the swipeable routes, we had `url` and `displayName`, now we even have `DISPLAY` and `EDIT` for the profile page.